### ISSUE TITLE
Error with harfbuzz

### ISSUE THREAD

**Issue by RCOBoxerman** [rdp/ffmpeg-windows-build-helpers#450]
Hi all,

I keep getting the following error: https://prnt.sc/s9wlrw

I've tried this several ways, does anyone know a good workaround?

**Comment by JohannesKauffmann** [comment]
I am getting this error as well on 1) WSL 18.04, 2) using a VM with 18.04 and 3) a VM with 20.04. I am using the 64 bit toolchain only.

For reference, [here](https://pastebin.com/vxsjLg1k) is the full console output of my 20.04 VM after running the script for the second time.

It seems like _hb-mutex.hh_ includes _hb.hh_ in the current directory, and _pthread.h_, _windows.h_ and _sched.h_ from the include path. Using locate they were all in
_sandbox/cross_compilers/mingw-w64-x86_64/x86_64-w64-mingw32/include/_
and
_sandbox/win64/harfbuzz_git/src/_

**Comment by JohannesMKauffmann** [comment]
Regarding commit [1fd3bd7](https://github.com/rdp/ffmpeg-windows-build-helpers/commit/1fd3bd7c694e6d51e5425eda08b6dba71e49ab84), it seems like Harfbuzz links correctly now. Or does it?
After deleting `sandbox/win64` and rerunning the script with the lastest version, it fails linking the Fontconfig executable `fc-cache.exe` while complaining about Harfbuzz libraries having undefined references to, again, [pthread_mutex](https://pastebin.com/pkZU5W32).
I also tried to pull the same trick (`export LDFLAGS=-lpthread` and `unset LDFLAGS`) at line 945 around generic_configure in build_libxml2() and line 966 around generic_configure in build_fontconfig(). This time linking `fc-cache.exe` is apparently succesfull, however linking `fc-validate.exe` is not. Same weird Harfbuzz undefined reference [yadayada](https://pastebin.com/rgUTHWTC).
I could not get it to go past linking `fc-validate.exe` even when linking freetype and zlib against pthread (since, one way or another, these libs all depend on each other). Let me know if I should try anything else.

**Comment by rdp** [comment]
Yeah something is weird in there, not sure what.  Try the latest!
-roger-

**Comment by darth2499** [comment]
Almost solved the issue. I got an error saying it could not find python, even though I had python3 installed. For users on Ubuntu 20.04, I ran `sudo apt install python-is-python3` and was able to proceed forward.

**Comment by rdp** [comment]
OK made a blind stab at it :)
